src/network/networkd-queue.c

```diff
@@ -63,7 +64,17 @@ static void request_free_object(RequestType type, void *object) {
         case REQUEST_TYPE_ROUTING_POLICY_RULE:
                 routing_policy_rule_free(object);
                 break;
-        case REQUEST_TYPE_SET_LINK:
+        case REQUEST_TYPE_SET_LINK_ADDRESS_GENERATION_MODE:
```

Comment:
It would be good to have a few lines in the commit message explaining _why_ it's a good idea to merge these two